config/purposes/heron_lthr_384.yml

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
Is there any point where they need extra labels in this pipeline? for QC or the extra tag plates or anything...

Reply:
Not that was mentioned in the requirements.